felayout_t3kit/dev/styles/main/contentElements/sliderContainer.less

```diff
@@ -23,6 +23,10 @@
     padding-right: 10px;
 }
 
+.slider-container__wrapper > div {
```

Comment:
What do you think if we will use `.slider-container__slide` instead of `div` tag?